evictiontest/workflow_cache_eviction_test.go

```diff
@@ -159,7 +159,7 @@ func (s *CacheEvictionSuite) TestResetStickyOnEviction() {
 	s.service.EXPECT().ResetStickyTaskQueue(gomock.Any(), gomock.Any()).DoAndReturn(mockResetStickyTaskQueue).Times(1)
 
 	client := internal.NewServiceClient(s.service, nil, internal.ClientOptions{})
-	workflowWorker := internal.NewAggregatedWorker(client, "taskqueue", worker.Options{})
+	workflowWorker := internal.NewAggregatedWorker(client, "taskqueue", worker.Options{LocalActivityWorkerOnly: true})
```

Comment:
This test requires workflow worker only and the mock service is setup accordingly. If we start activity worker, the mock would complain. The test works because this test does not register any activity and as side effect, activity worker does not start. With the change in this PR, activity worker would start and this test failed. Fix is to explicitly disable activity worker.